packages/ui/src/components/va-navbar/VaNavbar.vue

```diff
@@ -74,38 +74,46 @@ export default defineComponent({
 @import "variables";
 
 .va-navbar {
+  display: grid;
+  grid-template: "left center right" / 1fr 1fr 1fr;
```

Comment:
We should test that with different content sizes. Seems too rigid to me.

Reply:
I've checked all browsers and common resolutions, looks fine.